inlong-sdk/dataproxy-sdk/src/main/java/org/apache/inlong/sdk/dataproxy/config/ProxyConfigManager.java

```diff
@@ -632,7 +632,7 @@ private ProxyConfigEntry getProxyConfigEntry(DataProxyNodeResponse proxyCluster)
         if (ObjectUtils.isNotEmpty(proxyCluster.getIsSwitch())) {
             isSwitch = proxyCluster.getIsSwitch();
         }
-
+        int maxPacketLength = nodeList.get(0).getMaxPacketLength() != null ? nodeList.get(0).getMaxPacketLength() : -1;
```

Comment:
What to do if the 0th node in nodeList does not carry the value, but other nodes do, or the values ​​of each node are different?